nodebuilder/p2p/network.go

```diff
@@ -52,20 +52,20 @@ func (n Network) String() string {
 
 // networksList is a strict list of all known long-standing networks.
 var networksList = map[Network]struct{}{
-	Arabica:        {},
-	Mocha:          {},
-	BlockspaceRace: {},
-	Private:        {},
+	Mainnet: {},
```

Comment:
[discussion]
We may wanna call it just Celestia. We don't have `Testnet` and we have names for all other testnets.

Reply:
hmmmm, yes i think i agree with this

Reply:
p2p.Celestia

I think I _prefer_ `mainnet` but I'm fine with whatever. Not a big deal to me

Reply:
I like Mainnet more but am also not very opinionated. in the worst case we can change it later if we don't like it without breaking anything

Reply:
agree, don't feel very strong either way. BUT, with naming i always err on the side of "what do we say when we talk about it" and we're much more likely to say "on Arabica" or "if we are on Mainnet then we should remove the suffix" vs "if we are on Celestia". So....maybe just rolling with Mainnet is correct

Reply:
also as rene said, I feel better about p2p.Mainnet naming than p2p.Celestia, which might be confusing

Reply:
To me, it's similarly confusing as p2p.Arabica

Reply:
> "if we are on Celestia"

When we say on Celestia it should mean on mainnet by default IMO